src/js/node/tty.ts

```diff
@@ -4,6 +4,8 @@ const {
   getWindowSize: _getWindowSize,
 } = $cpp("ProcessBindingTTYWrap.cpp", "createBunTTYFunctions");
 
+const ERR_INVALID_ARG_TYPE = $zig("node_error_binding.zig", "ERR_INVALID_ARG_TYPE");
```

Comment:
We are creating this function several times. Might as well put it in a module and load from the cached copy.